src/js/entry.js

```diff
@@ -208,3 +219,24 @@ export function rootApp() {
         );
     }
 }
+
+export function noAccess() {
+    const { store } = spinUpStore();
+    const currPath = location.pathname;
+    const app = currPath.split('/').pop();
+    const userEntitlements = window.insights.chrome.auth.getUserEntitlements();
+    const apps = { insights: userEntitlements.insights.is_entitled };
```

Comment:
Please do not bind this only to `insights` essentially we want this to be applied to all apps not only settings. You get what user is entitled in the entitlements object and you can then check if path contains such app (it should be on 0th or 1st place, based on environment)

Reply:
@karelhala I am looking at the Jira for the issue, it sounds like we want this only in settings for now. Maybe I misunderstood here. Can you or @ryelo confirm that we are activating this everywhere?